_layouts/private.html

```diff
@@ -0,0 +1,105 @@
+              </li>
+              <li>
+                <span class="small">Archive as</span>
+                <a href="https://github.com/hyphacoop/organizing/new/master?filename=_posts/private/meeting-notes/2020-MM-DD-standup.md&value=Oops%21%20The%20notes%20from%20%2A%2A%7B%7B%20page.date%20%7C%20date%3A%20%22%25Y-%25m-%25d%22%20%7D%7D%20%7B%7B%20page.title%20%7D%7D%2A%2A%20is%20not%20open%20to%20public%20%3Alock%3A%0A%0AIf%20you%20have%20privileged%20access%2C%20you%20can%20%3Ca%20href%3D%22%7B%7B%20site.organizing-private%20%7D%7D%2F%7B%7B%20page.path%20%7C%20remove%3A%20%27_posts%2Fprivate%2F%27%20%7D%7D%22%3E%2A%2Aview%20the%20notes%20here%2A%2A%3C%2Fa%3E%20%3Akey%3A%0A%0APlease%20see%20the%20_Working%20Open_%20guidelines%20in%20our%20%5BHandbook%5D%28https%3A%2F%2Fhandbook.hypha.coop%2Fworking-open.html%29." target="_blank"><span class="small">publicly indexed</span></a>
+                <span class="small">with</span>
+                <a href="https://github.com/hyphacoop/organizing-private/new/master?filename=_posts/meeting-notes/2020-MM-DD-.md" target="_blank"><span class="small">restricted access</span></a>
+              </li>
+          </ul>
+
+          <ul>
+            <h3>Archived</h3>
+            {% for post in site.posts %}
+              {% if post.path contains "private/meeting-notes/" %}
+                <li>
+                  <a href="{{ site.baseurl }}{{ post.url }}">{{ post.date | date: "%Y-%m-%d" }} {{ post.title }}</a> :lock:
+                </li>
+              {% elsif post.path contains "meeting-notes/" %}
+                <li>
+                  <a href="{{ site.baseurl }}{{ post.url }}">{{ post.date | date: "%Y-%m-%d" }} {{ post.title }}</a>
+                </li>
+              {% endif %}
+            {% endfor %}
+          </ul>
+
+          <ul>
+            <h3>Informational Interviews</h3>
+            {% for post in site.posts %}
+              {% if post.path contains "private/informational-interviews/" %}
+                <li>
+                  <a href="{{ site.baseurl }}{{ post.url }}">{{ post.date | date: "%Y-%m-%d" }} {{ post.title }}</a> :lock:
+                </li>
+              {% elsif post.path contains "informational-interviews/" %}
+                <li>
+                  <a href="{{ site.baseurl }}{{ post.url }}">{{ post.date | date: "%Y-%m-%d" }} {{ post.title }}</a>
+                </li>
+              {% endif %}
+            {% endfor %}
+          </ul>
+
+          <ul>
+            <h3>2018 December Retreat</h3>
+            {% for post in site.posts %}
+              {% if post.path contains "2018-december-retreat/" %}
+                <li>
+                  <a href="{{ site.baseurl }}{{ post.url }}">{{ post.date | date: "%Y-%m-%d" }} {{ post.title }}</a>
+                </li>
+              {% endif %}
+            {% endfor %}
+          </ul>
+
+        </div>
```

Comment:
Now we gotta make this an `_includes` shared between the private and public layouts, then update all the templates with the long links, and we're good!

Let me know if you want me to take over this PR and do that. Thanks @patcon